src/Client/components/format-date-time.tsx

```
@@ -23,19 +23,16 @@ export function FormattedDateYear({ date }: {readonly date: string }) {
   });
 }
 
-export function FormattedTime({ date }: {readonly date: string }) { 
-
-  const openDate = new Date(date);
-  const formattedTime = openDate.toLocaleTimeString(undefined, {
+export function convertUtcToLocalTime(utcTimeString: string): string {
+  const utcDate = new Date(utcTimeString + 'Z');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)